README.md

```diff
@@ -32,20 +32,20 @@ $ git clone https://github.com/thingsapi/things.py && cd things.py && make insta
 
 ```python
 >>> import things
->>> things.tasks()
+>>> things.todos()
 [{'uuid': '2Ukg8I2nLukhyEM7wYiBeb',
   'title': 'Make reservation for dinner',
-  'type': 'task',
+  'type': 'to-do',
```

Comment:
In line 35 the method is called `todos`, here the type is called `to-do` - what about using the term `todo` everywhere just to be consistent?

Reply:
The terminology used in the app is "to-do" and "to-dos", so it seemed that was the consistent term to use. Python has limitations on what characters one can use, so I had to drop the minus for the function name.

It is unclear to me whether the limitations of Python should supersede the naming used in the app. Or... what do you think?